crates/refunder/src/arguments.rs

```diff
@@ -0,0 +1,40 @@
+use clap::Parser;
+use std::time::Duration;
+use url::Url;
+
+#[derive(Parser)]
+pub struct Arguments {
+    /// Minimum time an order must have been valid for, in order
+    /// to be eligble for refunding
+    #[clap(
+        long,
+        env,
+        default_value = "120",
+        value_parser = shared::arguments::duration_from_seconds,
+    )]
+    pub min_validity_duration: Duration,
+
+    /// Minimum slippage an order must have, in order
+    /// to be eligble for refunding
+    /// Front-end will place orders with a default slippage of 2% 
+    /// hence, we are requiring as a default 1.9%
```

Comment:
It's a bit implicit that the percentage needs to be encoded as a factor instead of the literal `1.9`.